tethys_cli/manage_commands.py

```diff
@@ -35,10 +38,13 @@ def add_manage_parser(subparsers):
     manage_parser.add_argument('-l', '--link', required=False, action='store_true',
                                help='Only used with collectstatic command. Link static directory to STATIC_ROOT '
                                     'instead of copying it. Not recommended.')
+    manage_parser.add_argument('--django-help', action='store_true',
+                               help='Display the help for specific scommands coming from Django. '
```

Comment:
"scommands" -> "commands"

Reply:
This is what I was trying too explain yesterday. "--help" calls the tethys help so I had to create the "--django-help" to avoid conflict and still be able to pull the help from django using tethys manage.